src/api/decommission-service/helpers/trigger-remove-workflows.js

```diff
@@ -13,18 +15,25 @@ import {
  * @param {import("pino").Logger} logger
  */
 async function triggerRemoveWorkflows(serviceName, repository, logger) {
+  logger.info(`Triggering remove workflows service: ${serviceName}`)
   const isTestSuite = repository.topics.includes('test-suite')
 
-  if (!isTestSuite) {
-    const zone = repository.topics.includes('backend') ? 'Protected' : 'Public'
+  if (isFeatureEnabled(featureToggles.removeServiceWorkflows)) {
+    if (!isTestSuite) {
+      const zone = repository.topics.includes('backend')
```

Comment:
might be better to use `src/api/deploy/helpers/lookup-tenant-service.js` instead of relying on topic, it returns a structure that has zone